ddtrace/tracer/tracer.go

```diff
@@ -171,9 +195,11 @@ func (t *tracer) worker() {
 			t.pushPayload(trace)
 
 		case <-ticker.C:
+			t.config.statsd.Incr("datadog.tracer.flush_count", []string{"reason:scheduled"}, 1)
```

Comment:
I kept pondering over the fact that this might be confusing in cases where flushing might fail because someone might think that it counts the number of flushes, when in fact it counts regardless of outcome, even for failures. Would it be better to call it `flush_triggered`? I'm unsure, just wondering...

Reply:
I think that's fair, especially given this comment by @cgilmour: https://github.com/DataDog/dd-trace-go/pull/534#discussion_r354036883
I'll change them to `flush_triggered`.